src/main/java/com/redhat/documentation/asciidoc/extraction/ExtractedModule.java

```diff
@@ -33,6 +34,12 @@ static boolean isNodeAModule(StructuralNode node) {
         return false;
     }
 
+    static String getFolder(Document doc) {
+        var foldernames=doc.getSourceLocation().getDir().split("/");
```

Comment:
This will blowup on Windows. Please use `File.separator` instead of "/".

Reply:
Actually, a better way to do this is probably creating a `File` from the `getDir()` call and getting the parent directory from that.